### ISSUE TITLE
Interpret e_bayescoring

### ISSUE THREAD

**Issue by JoshSchramm94** [markhwhiteii/bwstools#46]
Hey, 

I was wondering how I can actually interpret these bayes scoring? 
If I compare it to the ranks running HB Sawtooth I found a quite close match. However, Sawtooth also provides probability scores as well as zero-anchored interval scores. 
I thought that these bayes scores can be treated as raw logits scores and could easily be transformed to probability scores, either by the formula provided by Sawtooth (https://sawtoothsoftware.com/help/lighthouse-studio/manual/hid_web_maxdiff_hbv4.html?anchor=transformingweightsto0-100 , see "Transforming Weights to 0-100 Scale") or just running exp() and divide it by the sum(exp()) of all other item scores. However, when using these formulas the range is within 1 to 2% for all items, which seems odd. 

Thanks a lot for the help.
